app/src/main/java/org/sopt/and/SignInActivity.kt

```diff
@@ -0,0 +1,318 @@
+        )
+
+        Spacer(modifier = Modifier.size(35.dp))
+    }
+}
+
+@Composable
+fun InputField(value: MutableState<String>, placeholder: String, onValueChange: (String) -> Unit) {
+    TextField(
+        value = value.value,
+        onValueChange = { newText -> value.value = newText; onValueChange(newText) },
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(60.dp),
+        placeholder = { Text(text = placeholder, color = Color.Gray) },
+        colors = TextFieldDefaults.colors(
+            focusedContainerColor = Color.DarkGray,
+            unfocusedContainerColor = Color.DarkGray,
+            focusedTextColor = Color.White,
+            unfocusedTextColor = Color.White,
+            focusedIndicatorColor = Color.Transparent,
+            unfocusedIndicatorColor = Color.Transparent
+        ),
+        shape = RoundedCornerShape(5.dp),
+        singleLine = true
+    )
+}
+
+@Composable
+fun PasswordField(
+    value: String,
+    onValueChange: (String) -> Unit,
+    showPassword: Boolean,
+    onVisibilityChange: () -> Unit
+) {
+    TextField(
+        value = value,
+        onValueChange = onValueChange,
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(60.dp),
+        placeholder = { Text(text = "비밀번호", color = Color.Gray) },
+        colors = TextFieldDefaults.colors(
+            focusedContainerColor = Color.DarkGray,
+            unfocusedContainerColor = Color.DarkGray,
+            focusedTextColor = Color.White,
+            unfocusedTextColor = Color.White,
+            focusedIndicatorColor = Color.Transparent,
+            unfocusedIndicatorColor = Color.Transparent
+        ),
+        shape = RoundedCornerShape(5.dp),
+        visualTransformation = if (showPassword) VisualTransformation.None else PasswordVisualTransformation(),
+        trailingIcon = {
+            Text(
+                text = if (showPassword) "Hide" else "Show",
+                color = Color.White,
+                modifier = Modifier.clickable { onVisibilityChange() }
+            )
+        },
+        singleLine = true
+    )
+}
+
+@Composable
+fun SignInButton(text: String, onClick: () -> Unit) {
+    Button(
+        onClick = onClick,
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(50.dp),
+        colors = ButtonDefaults.buttonColors(
+            containerColor = Color.Blue,
+            contentColor = Color.White
+        )
+    ) {
+        Text(text = text, fontSize = 15.sp)
+    }
+}
+
+@Composable
+fun ActionLinks(onSignUpClick: () -> Unit) {
+    Row(
+        modifier = Modifier.fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceEvenly
+    ) {
+        ActionText(text = "아이디 찾기", onClick = { })
+        Text(text = "|", color = Color.Gray)
+        ActionText(text = "비밀번호 재설정", onClick = { })
+        Text(text = "|", color = Color.Gray)
+        ActionText(text = "회원 가입", onClick = onSignUpClick)
```

Comment:
회원 가입을 제외한 텍스트들에는 onClick을 주고싶지 않다면 이렇게 써보는것도 깔끔할 것 같습니다!!
```suggestion
        Text(text = "아이디 찾기 | 비밀번호 재설정 | ", color = Color.Gray)
        ActionText(text = "회원 가입", onClick = onSignUpClick)
```

Reply:
아하..! 나중에 기능이 들어갈까 해서 넣어놨는데 나중에 고치는게 더 좋아보이네요